options/linux-headers/include/linux/cdrom.h

```diff
@@ -73,6 +86,55 @@ struct cdrom_generic_command {
 	void *reserved[1];
 };
 
+typedef struct {
+	// __be16 ?
```

Comment:
Why not use standard stdint integers in this struct?

Reply:
As this struct is directly from the Linux kernel, I figured to not change it at all where possible. I saw this behaviour in other places too, which is why I decided against replacing them.

Reply:
Afaict, they are just type specifiers, so they don't mean anything at runtime. But at compile time they enforce that you only ever assign a big endian value to it. It's probably really not needed in mlibc, so I'd say to just convert them to regular stdints.

Reply:
Alright, I've replaced them with the normal kernel types.

Reply:
Do we use ``__ux`` anywhere else in mlibc? Why not just use ``uintx_t``?